include/mesh.hpp

```diff
@@ -17,7 +17,8 @@ class Mesh;
 
 class Mesh {
 public:
-  Mesh(int nintervals = 10, double dt = 0.1, int nt = 1000);
+  Mesh();
+  Mesh(int nintervals, double dt = 0.1, int nt = 1000);
```

Comment:
This causes regressions. There were previously calls made to the `Mesh()` constructor with no arguments, expecting it to be equivalent to a call of `Mesh(10, 0.1, 1000)`. However, now it will call your newly-defined default constructor, which uses a different value for `nintervals_in` and doesn't perform any of the setup done in the body of the original constructor.